### DISCUSSION TITLE
how to close the upgrade notifications in docker

### DISCUSSION THREAD

**Discussion by huang403922595** [netbox-community/netbox-docker#741] [Getting Help] [answered]
I have tried to change the env/netbox.env .

For example, I tried to change 'RELEASE_CHECK_URL' in the netbox.env ,just like this:

1. #RELEASE_CHECK_URL=https://api.github.com/repos/netbox-community/netbox/releases
Result: Do not work. Still notifying to upgrade.

2.RELEASE_CHECK_URL=None
Result: After 'docker-compose up -d',the container 'netbox-docker-netbox-worker-1' and 'netbox-docker-netbox-1' will be down.

So i do not know how to close the upgrade notifications in docker now.

**Comment by Fyne5** [comment, marked as answer]
I think "None" is not legal, use "False" instead

**Reply by huang403922595** [reply]
I try the 'False'. The result is the same as the 'None'.
Then I set  'RELEASE_CHECK_URL='.  No 'None' or 'False'. It seems work.
